### ISSUE TITLE
Messages Update

### ISSUE THREAD

**Issue by vulfgang** [zvakanaka/church-bulk-text#1]
The bulk texter no longer works since the recent update 02/2023
Unsure if this has to do with the switch to RCS over SMS or if they patched out the extension but the texter won't send

**Comment by zvakanaka** [comment]
I've looked into it for about an hour and started messing with [recording mouse events, then playing them back](https://codepen.io/zvakanaka/pen/YzOGRpv) (with dispatchEvent), but haven't solved it yet. I am very busy this year, but will occasionally look into it. Any help would be very much appreciated!

**Comment by brismuth** [comment]
@zvakanaka I saw your email about this issue.

For [Bulk Texter Pro](https://www.bulktexterpro.com/) I needed to update the click handling in order to integrate with Google Photos to send images. `element.click()` didn't work there either, but this did.

This StackOverflow post is how I did it:
https://stackoverflow.com/a/55068681/1569320

```js
var simulateMouseEvent = function(element, eventName, coordX, coordY) {
  element.dispatchEvent(new MouseEvent(eventName, {
    view: window,
    bubbles: true,
    cancelable: true,
    clientX: coordX,
    clientY: coordY,
    button: 0
  }));
};

var theButton = document.querySelector('ul form button');

var box = theButton.getBoundingClientRect(),
        coordX = box.left + (box.right - box.left) / 2,
        coordY = box.top + (box.bottom - box.top) / 2;

simulateMouseEvent (theButton, "mousedown", coordX, coordY);
simulateMouseEvent (theButton, "mouseup", coordX, coordY);
simulateMouseEvent (theButton, "click", coordX, coordY);
```

Hopefully this does the trick for you, and if not, I'm also planning to integrate Bulk Texter Pro with Google Messages at some point in the future as well, so I'll need to solve this either way and I'll get back to you. :)

**Comment by vulfgang** [comment]
Awesome thanks, I had a working version with google messages that has recently been patched out. Haven't gotten back to investigate the changes as of yet

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by brismuth** [comment]
@zvakanaka I'm now releasing a Google Messages / Android integration for [Bulk Texter Pro](https://www.bulktexterpro.com/), and as part of that I solved this issue. 

I've submitted a fix in PR #2. Let me know if you have any questions – feel free to email me.
